src/pages/variantHistory/ColumnHeaders.tsx

```diff
@@ -1,17 +1,27 @@
 import styled from "@emotion/styled";
-import { Skeleton } from "antd";
 import { context, Cell } from "components/HistoryTable";
-import { array } from "utils";
+import { getTaskHistoryRoute } from "constants/routes";
+import { array, string } from "utils";
 
 const { mapStringArrayToObject } = array;
+const { trimMiddleText } = string;
 const { useHistoryTable } = context;
-const { HeaderCell } = Cell;
+const { LoadingCell, ColumnHeaderCell } = Cell;
+
+const maxLength = 50;
+const trailingLength = 10;
```

Comment:
I decided these values arbitrarily for both pages ... 🙂 Open to hearing other ways to trim the name